### DISCUSSION TITLE
links captured are displayed as [caption](url) rather than clickable caption

### DISCUSSION THREAD

**Discussion by happy15** [EINDEX/logseq-copilot#45] [Q&A] [answered]
Take following part in https://blog.promaton.com/how-to-avoid-kpi-psychosis-in-your-organization-5ffc83967f2b as an example:
>Just to give a glimpse: we take information that is easiest to gather ([availability bias](https://en.wikipedia.org/wiki/Availability_heuristic)), often that is heavily affected by the latest info we got ([recency bias](https://en.wikipedia.org/wiki/Recency_bias)) and when we make conclusions we tend to overgeneralize ([halo/horn effect](https://en.wikipedia.org/wiki/Horn_effect)) with more confidence than it was justified by our lintels ([overconfidence bias](https://en.wikipedia.org/wiki/Overconfidence_effect)). On top, we are typically stubborn and seldom change our opinions and worldview ([anchoring bias](https://en.wikipedia.org/wiki/Anchoring_%28cognitive_bias%29),[ confirmation bias](https://en.wikipedia.org/wiki/Confirmation_bias)). This is especially problematic when changes happen slowly and in small steps. For instance, the way software development productivity changes in our team.

When captured to logseq, the links are displayed as `[caption](url)`, because they are escaped as `\[availability bias]\(https://en.wikipedia.org/wiki/Availability_heuristic)`. 

Any solution or workaround to this? Thanks.

**Comment by EINDEX** [comment]
no, actually, this is a feature, I want to avoid content injection to logseq.

But I missing this case. 😢

**Comment by EINDEX** [comment, marked as answer]
I have fixed this. Currently link will be a link

**Reply by happy15** [reply]
good to know, will try.
